src/test/ui/parser/issue-104867-inc-dec.rs

```diff
@@ -0,0 +1,30 @@
+struct S {
+    x: i32,
+}
+
+fn test1() {
+    let mut i = 0;
+    i++; //~ ERROR Rust has no postfix increment operator
+}
+
+fn test2() {
+    let s = S { x: 0 };
+    s.x++; //~ ERROR Rust has no postfix increment operator
+}
+
+fn test3() {
+    let mut i = 0;
+    if i++ == 1 {} //~ ERROR Rust has no postfix increment operator
+}
+
+fn test4() {
+    let mut i = 0;
+    ++i; //~ ERROR Rust has no prefix increment operator
+}
+
+fn test5() {
+    let mut i = 0;
+    if ++i == 1 { } //~ ERROR Rust has no prefix increment operator
+}
+
```

Comment:
Can we add a test for

```rust
let mut i = 0;
loop { break; }
i++;
loop { break; }
++i;
```
and one for
```rust
let mut i = 0;
let _ = i + ++i;
let _ = ++i + i;
let _ = ++i + ++i;
let _ = i + i++;
let _ = i++ + i;
let _ = i++ + i++;
let _ = ++i + i++;
let _ = i++ + ++i;
```

(that last one might be problematic and need its own test file)